doc/nrf/libraries/bluetooth_services/enocean.rst

```diff
@@ -49,21 +49,21 @@ The Bluetooth mesh stack does this automatically, so when combined, the applicat
 Commissioning
 *************
 
-Before the library is able to process sensor or button messages from nearby EnOcean devices, the devices must be commissioned.
+The devices must be commisioned to enable the library to process sensor or button messages from nearby EnOcean devices.
```

Comment:
```suggestion
The devices must be commissioned so that the library can process sensor or button messages from nearby EnOcean devices.
```
Typo in "commissioned", and I think this structure is better than the "to.. to.."